### DISCUSSION TITLE
How do I re-index nextra's flexsearch? I am seeing stale results

### DISCUSSION THREAD

**Discussion by dionysuzx** [shuding/nextra#1266] [Q&A]
When I use nextra's flexsearch I am seeing results for markdown pages which no longer exist (because I re-organized the folder structure. how do i remove these results / reset flexsearch to re-index the current structure / documents? 

Through trying this on safari and firefox i've realized that this index is local to the browser, because it differs based on the browser i used. Isn't this a bug? it's holding on to stale results for users. Should there be a way for the website to instruct that if it is a new build it should flush the users search cache?

cc @shuding

**Comment by AKRking** [comment]
whats the current status,were u able to fix it?

**Comment by smythp** [comment]
Did anyone find a solution for this?
